solana/fuzz/src/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
can you check in the accompanying Cargo.lock file too?

Reply:
Done!